File: __tests__/__fixtures__/output/node/axios/jsonObj-null-value.js
var axios = require("axios").default;

var options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  headers: {'content-type': 'application/json'},
  data: {foo: null}
};

axios.request(options).then(function (response) {
  console.log(response.data);
}).catch(function (error) {
  console.error(error);
});

File: test/fixtures/output/python/python3/jsonObj-null-value.py
import http.client

conn = http.client.HTTPSConnection("httpbin.org")

payload = "{\"foo\":null}"

headers = { 'content-type': "application/json" }

conn.request("POST", "/anything", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
